src/linear_algebra/basis.lean

```diff
@@ -494,8 +495,31 @@ begin
   simp [constr_basis hC, right_inverse_inv_fun (linear_map.range_eq_top.1 hf_surj) c]
 end
 
+/-- Any linear map `f : p →ₗ[K] V'` defined on a subspace `p` can be extended to the whole
+space. -/
+lemma linear_map.exists_extend {p : submodule K V} (f : p →ₗ[K] V') :
```

Comment:
@JasonKYi, this looks quite similar to some stuff in your PR for linear_algrbra.projections.

Reply:
(#5981, presumably)

Reply:
I can use `of_is_comp` here but the proof will not be much shorter.